### ISSUE TITLE
run and run-stages commands

### ISSUE THREAD

**Issue by jacintoArias** [demosense/demorepo#10]
I see two different use cases for demorepo.

First is orchestating the different stages of a CI and CD pipeline. For that we can run a `demorepo run --stage <stage>` command and it will run and log the appropiate stage for each target project.

However in particular scenarios we may want to execute a pipeline that needs various pre and post operations for every other project, e.g. provisioning resources for an integration test.

With the current design, we must define a "multi stage approach" (the one I am currently using):

```
demorepo run --stage provision
demorepo run --stage test-integrate
demorepo run --stage teardown
```

This would indeed work, but in my opinion *provision* and *teardown* should not be defined as stages. An alternative would consist on:

Create two commands, i.e. `demorepo run <command>` and `demorepo run-stage <stage>` to allow running generic commands or pipeline stages defined in the `demorepo.yml` files. This `demorepo run` command is generic enough to also allow some debugging and local testing capabilities in our demorepos. In addition, this does not break the previous workflow, so it is not a breaking change.

**Comment by jacintoArias** [comment]
Shall we close this @jcozar? I was resolved by PM #13

**Comment by jcozar87** [comment]
Yes we should. Closed by PM #13
